gradio/components/highlighted_text.py

```diff
@@ -49,6 +45,7 @@ def __init__(
         visible: bool = True,
         elem_id: str | None = None,
         elem_classes: list[str] | str | None = None,
+        interactive: bool = False,
```

Comment:
> Theres a bug right now which means we're not detecting whether the component is an input or output, so you have to use Interactive=True

Set the default value to None and Blocks.svelte will set it to be interactive if it's used as input to an event!